### PR TITLE
Edit userlist

### PR DESCRIPTION
**Pull request by zihaoxu20010109**
## An edit user list ui for a user's anime list
- When user clicks on List Settings a modal pops up to update the UserList settings
- Able to update the name
- Able to update the rating system

https://user-images.githubusercontent.com/59790644/144687466-3d317a91-b35b-4868-b1e9-56df76771921.mp4

em

### CODE REVIEW

File: client/src/components/list/_UserList.tsx [hunk 1]
```diff
@@ -28,35 +30,114 @@ export interface IListItem {
...
+    }
+  });
+  
+  return (
+    <Modal isCentered scrollBehavior="inside" isOpen={isOpen} onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent padding="10px" maxW="56rem">
+        <ModalCloseButton />
+        <ModalHeader>
+          <Heading>update name and rating system</Heading>
```
Comment (jvoucsb) [inline client/src/components/list/_UserList.tsx:99]: Change header to `List Settings`

File: client/src/components/list/_UserList.tsx [hunk 2]
```diff
@@ -28,35 +30,114 @@ export interface IListItem {
...
+        </ModalHeader>
+        <form onSubmit={formik.handleSubmit}>
+          <ModalBody pb={6}>
+            <FormControl>
+              <FormLabel>New name</FormLabel>
```
Comment (jvoucsb) [inline client/src/components/list/_UserList.tsx:104]: Change to just `Name`

File: client/src/components/list/_UserList.tsx [hunk 3]
```diff
@@ -28,35 +30,114 @@ export interface IListItem {
...
+              <Input id="name" {...formik.getFieldProps("name")} />
+            </FormControl>
+
+            <FormControl mt={4}>
+              <FormLabel>New ratingsystem</FormLabel>
```
Comment (jvoucsb) [inline client/src/components/list/_UserList.tsx:109]: Change to just `Rating System`

**Review by jvoucsb — CHANGES_REQUESTED** [review]

**Review by rowantran — APPROVED** [review]
code looks good, tested locally as well and everything looks good! nice work

**Review by christopherbarnett — APPROVED** [review]
cb - lgtm

**Pull request merged**